File: IsometricShooter/Scripts/CS/GUI/CrosshairGui.cs
using UnityEngine;
using System.Collections;

public class CrosshairGui : MonoBehaviour {

	public Camera playerCamera; //drop camera of player here
	public Transform playerAim; //drop aim-Transform of player here
	public GUITexture crosshair;

	// Use this for initialization
	void Start () {
	
		Cursor.visible = false; //hide mouse cursor
	
	}
	
	void OnGUI () {

		crosshair.transform.position = playerCamera.WorldToViewportPoint(playerAim.position);

	}
	
}

File: IsometricShooter/Scripts/JS/GUI/CrosshairGui.js
#pragma strict

var playerCamera : Camera; //drop camera of player here
var playerAim : Transform; //drop aim-Transform of player here
var crosshair : GUITexture;

function Start () {

	Cursor.visible = false; //hide mouse cursor

}

function OnGUI () {

	crosshair.transform.position = playerCamera.WorldToViewportPoint(playerAim.position);

}